cli/bpmetadata/cmd.go

```diff
@@ -53,8 +54,35 @@ func generate(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("error getting working dir: %w", err)
 	}
 
-	bpPath := path.Join(wdPath, mdFlags.path)
+	var allBpPaths []string
+	currBpPath := path.Join(wdPath, mdFlags.path)
+	allBpPaths = append(allBpPaths, currBpPath)
+
+	// if nested, check if modules/ exists and create paths
+	// for submodules
+	if mdFlags.nested {
+		modulesPathforBp := path.Join(currBpPath, modulesPath)
+		_, err = os.Stat(modulesPathforBp)
+		if err == nil {
```

Comment:
nit: We should check if os.IsNotExist. Other errors we can surface. If !os.IsNotExist, we can add an info log that no sub modules found.

Reply:
ack